ft_datasets/__init__.py

```diff
@@ -3,4 +3,5 @@
 
 from .grammar_dataset import get_dataset as get_grammar_dataset
 from .alpaca_dataset import InstructionDataset as get_alpaca_dataset
-from .samsum_dataset import get_preprocessed_samsum as get_samsum_dataset
\ No newline at end of file
+from .samsum_dataset import get_preprocessed_samsum as get_samsum_dataset
+from .completion_dataset import get_preprocessed_completion as get_completion_dataset
```

Comment:
Can we rename completion to code completion to make it more obvious to the user?

Reply:
it's not code completion, it's just corpus text completion, it's what I use for pretraining from textbooks for example